### PR TITLE
Fix for swagger docs serving

### PR DESCRIPTION
**Pull request by dssei**
## Describe your changes and provide context
Swagger documentation should automatically be registered if `api.swagger` setting in `app.toml` is enabled
```toml
swagger = true
```
It was broken though, as this configuration was ignored in the app. Simply adding route would not work either. As just enabling it would pull swagger that is generated in `sei-cosmos` only lacking some modules APIs.
Below solution adds similar capability to `sei-chain` level and is tweaked to involve `ignite` generated swagger API docs. 

This change:
- adds swagger route (if configured)
- updates the path openapi/swagger yml is generated
- adds swagger-ui template
- introduces similar way of serving docs (with `statik` lib) to that of cosmos sdk (note, that we are not reusing the same scripts though, as in this repo we use `ignite` for openapi/swagger docs generation.
- updates README

## Testing performed to validate your change
- unit test for adding swagger route
- deployed and tested on local node

### CODE REVIEW

File: app/app_test.go [hunk 1]
```diff
@@ -405,3 +411,70 @@ func TestDecodeTransactionsConcurrently(t *testing.T) {
 	require.Nil(t, typedTxs[1])
 	require.NotNil(t, typedTxs[2])
 }
+
+func TestApp_RegisterAPIRoutes(t *testing.T) {
```
Comment (yzang2019) [inline app/app_test.go:415]: Nice! Thanks for adding the test

File: scripts/update-swagger-ui-statik.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (yzang2019) [inline scripts/update-swagger-ui-statik.sh:1]: When should this script be used? Do you mind adding some instructions?

**Review by yzang2019 — APPROVED** [review]

File: scripts/update-swagger-ui-statik.sh (hunk 2 above)
Comment (dssei) [reply scripts/update-swagger-ui-statik.sh:1]: Anytime when we modify endpoints/related protocol buffer. I will make it more explicit in readme

Comment (dssei) [reply scripts/update-swagger-ui-statik.sh:1]: Done

**Pull request merged**